bin/install_uninstall.ml

```diff
@@ -10,22 +10,33 @@ let interpret_destdir ~destdir path =
   | Some destdir -> Path.append_local destdir (Path.local_part path)
 
 let get_dirs context ~prefix_from_command_line ~from_command_line =
-  let open Fiber.O in
+  let open Dune_engine.Install.Section.Paths.Roots in
```

Comment:
Instead of opening this module, could you alias it as such:

```
module Roots = Dune_engine.Install.Section.Paths.Roots
```

The reader wouldn't need to think about where functions such as `map` are coming from.